### PR TITLE
Cleanup for code renovations

### PR DESCRIPTION
**Pull request by JamesVStone**
All the changes I made to integrate with a discord bot, without the discord bot!

This pretty much is just adding some comments and other readability improvements, added some functions to decrease code repetition etc..

Basically, a good starting point for some major renovations.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,4 +3,5 @@ Flask-Limiter
 GitPython
 flask_sqlalchemy
 swaglyrics
-requests
\ No newline at end of file
+requests
+discord
```
Comment (JamesVStone) [inline requirements.txt]: remove discord dependency

**Comment by aadibajpai** [comment]
> Hello @JamesVStone! Thanks for updating this PR.
> 
> * In the file [`issue_maker.py`](https://github.com/SwagLyrics/swaglyrics-backend/blob/ec6d89624d6e4c73df53610fce2782a47b436e2d/issue_maker.py):
> 
> > [Line 54:77](https://github.com/SwagLyrics/swaglyrics-backend/blob/ec6d89624d6e4c73df53610fce2782a47b436e2d/issue_maker.py#L54): [E126](https://duckduckgo.com/?q=pep8%20E126) continuation line over-indented for hanging indent
> > [Line 354:16](https://github.com/SwagLyrics/swaglyrics-backend/blob/ec6d89624d6e4c73df53610fce2782a47b436e2d/issue_maker.py#L354): [E203](https://duckduckgo.com/?q=pep8%20E203) whitespace before ':'
> > [Line 354:17](https://github.com/SwagLyrics/swaglyrics-backend/blob/ec6d89624d6e4c73df53610fce2782a47b436e2d/issue_maker.py#L354): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ':'
> > [Line 354:17](https://github.com/SwagLyrics/swaglyrics-backend/blob/ec6d89624d6e4c73df53610fce2782a47b436e2d/issue_maker.py#L354): [E999](https://duckduckgo.com/?q=pep8%20E999) SyntaxError: invalid syntax
> > [Line 354:19](https://github.com/SwagLyrics/swaglyrics-backend/blob/ec6d89624d6e4c73df53610fce2782a47b436e2d/issue_maker.py#L354): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> 
> ##### Comment last updated at 2020-01-03 19:47:43 UTC

@JamesVStone take care of this please :)

File: issue_maker.py [hunk 2]
```diff
@@ -36,26 +41,31 @@
...
 # artist and song regex
 asrg = re.compile(r'[A-Za-z\s]+')
 
-SQLALCHEMY_DATABASE_URI = "mysql+mysqlconnector://{username}:{password}@{username}.mysql.pythonanywhere-services." \
-                          "com/{username}${databasename}".format(
-                                                                username=username,
-                                                                password=os.environ['DB_PWD'],
-                                                                databasename="strippers",
-                                                            )
+# SQLAlchemy configuration
```
Comment (aadibajpai) [inline issue_maker.py]: I think I preferred the previous style

File: issue_maker.py [hunk 3]
```diff
@@ -325,52 +337,71 @@ def delete_line():
...
 
 
+"""
+`github_webhook` function handles all notification from GitHub relating to the org. Documentation for the webhooks can
+be found at https://developer.github.com/webhooks/
+"""
+
+
+# helper function
+def validate_request(req):
```
Comment (aadibajpai) [inline issue_maker.py]: Maybe you should add this to utils.py then? That's the file for helper functions

Comment (JamesVStone) [reply issue_maker.py]: sure

**Comment by aadibajpai** [comment]
Oh hey, so this is ready right?

**Comment by JamesVStone** [comment]
yup, sure is

**Comment by aadibajpai** [comment]
Merging, hope nothing breaks

**Pull request merged**